corehq/apps/app_manager/tests/data/suite/reports_module_data_detail.xml

```diff
@@ -1,14 +1,14 @@
 <partial>
-  <detail id="reports.-1044519270389493083.data">
+  <detail id="reports.d3ff18cd83adf4550b35db8d391f6008-None.data">
```

Comment:
Can the counter start at zero? Having the string "None" in an id says "bug" to anyone unfamiliar with this code.